src/components/Label/Label.tsx

```diff
@@ -181,8 +181,14 @@ const StyledTextSpan = styled.span<IStyledLabel>`
 `
 
 const Label: ILabel = props => {
-  const { className, style, ...rest } = props
-  const { text, icons, insetColor } = rest
+
+  const labelPropKeys = ["size", "rounded", "outlined", "icons", "focus",
```

Comment:
Would be nice If there is a way to derive this list from the type.

Reply:
Yes, it would be nice.